.github/workflows/release.js.yml

```diff
@@ -27,6 +27,10 @@ jobs:
       - name: Unit tests
         run: yarn test
 
+      # Build the project
+      - name: Build
+        run: yarn build
```

Comment:
Added `yarn build` into prepare script. I looked into one-sdk and there we use `prepack` script. From the docs `prepare` looks more appropriate.

Reply:
Sorry for confusion, prepack is the correct option here since we use Yarn 1: https://yarnpkg.com/advanced/lifecycle-scripts

With this lifecycle script you can remove explicit calls to yarn build to avoid building the package twice.